src/lib/token-utils.js

```diff
@@ -40,7 +40,7 @@ const KNOWN_TOKENS_FALLBACK = new Map([
   ],
 ])
 
-const LOCAL_TOKEN_ICONS = new Map([['TESTTEC', tokenIconSvg]])
+const LOCAL_TOKEN_ICONS = new Map([['xDAI', tokenIconSvg]])
```

Comment:
maybe we could add support to xDai, but not remove TESTEC from the array? 

the function `getTokenIconBySymbol()` will return the token requested according to the symbol passed